packages/commons/src/events/FormConfig.ts

```diff
@@ -15,7 +15,10 @@ import { TranslationConfig } from './TranslationConfig'
 export const FormPage = z.object({
   id: z.string().describe('Unique identifier for the page'),
   title: TranslationConfig.describe('Header title of the page'),
-  fields: z.array(FieldConfig).describe('Fields to be rendered on the page')
+  fields: z.array(FieldConfig).describe('Fields to be rendered on the page'),
+  // At first we tried to hide pages on review if they have no fields to show.
+  // But in some cases, e.g. birth declaration supporting documents, we want to show the page even if no documents have been added.
+  hideOnReview: z.boolean().default(false).optional()
```

Comment:
👍 Good point. What do you think about flipping the condition to be `alwaysShowOnReview` etc to keep the configuration experience so you don't in most cases have to worry about setting up the flag. So

Introduction page with only non-input fields = hides by default
Supporting documents page = while configuring I notice it now doesn't show in review, which triggers me to search for this configuration flag

While I was writing this I also started thinking if "FILE" is also a bit of a special case in this context as we show those fields on the review page, only we move them to the file selector. 

If you had logic like "If all inputs are Output = null and there are no file inputs THEN hide the section completely", you'd solve 100% of currently known cases without extra config flags